### PR TITLE
Separate the RadMon's transfer script from image generation.

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
This moves the actual file transfer into a new script that is called both by `image_gen/ush/RadMon_IG_glb.sh` and a new script `image_gen/RunTransfer.sh`.  It's sometimes useful to be able to submit the transfer job as a one-off and these changes make that possible.

Note that file transfers will only work from wcoss2 and require the user to have a password free login to the web server.

Ref #50 .

### CODE REVIEW

File: src/Radiance_Monitor/image_gen/ush/transfer.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ ${IMGNDIR} != "/" ]]; then
+   if [[ $MY_MACHINE = "wcoss2" ]]; then
```
Comment (DavidHuber-NOAA) [inline src/Radiance_Monitor/image_gen/ush/transfer.sh]: Would it make more sense to move the wcoss2 check outside of this job, perhaps to RadMon_IG_glb.sh, so a job doesn't have to be submitted for other systems?

**Review by DavidHuber-NOAA — CHANGES_REQUESTED** [review]
I'm not sure why RunTransfer.sh was created.  It doesn't seem to be called anywhere.  Should it be deleted or should it be called instead of transfer.sh?  If the latter, then should transfer.sh be deleted?

**Comment by EdwardSafford-NOAA** [comment]
I'm sorry I didn't fully describe my reasoning for the `RunTransfer.sh` script.  It isn't called by any of the other scripts.  I use that myself when I need to run a transfer without running image generation.  That doesn't happen routinely, but I do use it occasionally to update the server after prod switches or when the regular transfers get interrupted.

File: src/Radiance_Monitor/image_gen/ush/transfer.sh (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply src/Radiance_Monitor/image_gen/ush/transfer.sh]: Fair point.  I'll make that change.

**Comment by DavidHuber-NOAA** [comment]
@EdwardSafford-NOAA That makes sense RE RunTransfer.sh.  Thanks for clarifying.

File: src/Radiance_Monitor/image_gen/ush/RadMon_IG_glb.sh [hunk 2]
```diff
@@ -348,14 +343,11 @@ if [[ $RUN_TRANSFER -eq 1 ]]; then
...
-      job="${IG_SCRIPTS}/Transfer.sh --nosrc ${RADMON_SUFFIX}"
-
       export WEBDIR=${WEBDIR}/${RADMON_SUFFIX}/pngs
 
       cmdfile="${PLOT_WORK_DIR}/transfer_cmd"
-      echo "${IG_SCRIPTS}/Transfer.sh --nosrc ${RADMON_SUFFIX}" >$cmdfile
+      echo "${IG_SCRIPTS}/transfer.sh" >$cmdfile
       chmod 755 $cmdfile
 
       run_time="$rhr$cmin"	# HHMM format for qsub
```
Comment (DavidHuber-NOAA) [inline src/Radiance_Monitor/image_gen/ush/RadMon_IG_glb.sh:353]: So should transfer.sh be run from all systems, then, or just wcoss2?

Comment (EdwardSafford-NOAA) [reply src/Radiance_Monitor/image_gen/ush/RadMon_IG_glb.sh:353]: Just wcoss2.  There's a preexisting check at line 327 (not in view) for that.

Comment (DavidHuber-NOAA) [reply src/Radiance_Monitor/image_gen/ush/RadMon_IG_glb.sh:353]: Ah, got it.

**Review by DavidHuber-NOAA — APPROVED** [review]

**Pull request merged**